configure

```diff
@@ -375,6 +379,31 @@ parser.add_option('--enable-static',
     dest='enable_static',
     help='build as static library')
 
+parser.add_option('--static-libstdc++',
+    action='store_true',
+    dest='use_static_libstdcpp',
+    help='statically link to libstdc++ library instead of dynamically linking')
```

Comment:
Looks like this was removed in v6.x so thinking we don't want to add in v4.x.  I also don't see it in https://github.com/nodejs/node/pull/6994 so I'm wondering where it came from.